sdk/resource/env.go

```diff
@@ -17,14 +17,20 @@ package resource // import "go.opentelemetry.io/otel/sdk/resource"
 import (
 	"context"
 	"fmt"
+	"go.opentelemetry.io/otel/semconv"
```

Comment:
Looks like gofmt doesn't like this in this location.

Reply:
GoLand does some things well, but collapsing the import stanza by default is really annoying...